src/app/collective-rewards/rewards/components/Metrics/MetricsCard.tsx

```diff
@@ -88,4 +88,22 @@ export const MetricsCardTitle: FC<{ title: string; 'data-testid': string }> = ({
   </Typography>
 )
 
+type MetricsCardContentProps = {
+  children: ReactNode
+}
+
+export const MetricsCardContent: FC<MetricsCardContentProps> = ({ children }) => (
```

Comment:
This component is use for the `AllTimeShare`, there are some re-usable from this component but Im not sure if this component should be here, if I need to rename it or something like that.

Reply:
For now, I think it's fine to have it there 👍🏼